website/docs/fabric.md

```diff
@@ -0,0 +1,128 @@
+However, since the table is in ADLS, you need to update datasets path and hadoop configurations.
+
+For e.g. if the source table is in Iceberg format, the configuration file should look like:
+
+```yaml md title="my_config.yaml"
+sourceFormat: ICEBERG
+targetFormats:
+  - DELTA
+datasets:
+  -
+    tableBasePath: abfs://mycontainer@mystorageaccount.dfs.core.windows.net/default/people
+    tableDataPath: abfs://mycontainer@mystorageaccount.dfs.core.windows.net/default/people/data
+    tableName: people
+    # In the configuration above `default` refers to the default spark database.
+```
+
+An example hadoop configuration for authenticating to ADLS storage account is as follows:
+```xml md title="hadoop.xml"
+<configuration>
+  <property>
+    <name>fs.azure.account.auth.type</name>
+    <value>OAuth</value>
+  </property>
+  <property>
+    <name>fs.azure.account.oauth.provider.type</name>
+    <value>org.apache.hadoop.fs.azurebfs.oauth2.ClientCredsTokenProvider</value>
+  </property>
+  <property>
+    <name>fs.azure.account.oauth2.client.endpoint</name>
+    <value>https://login.microsoftonline.com/[tenant-id]/oauth2/token</value>
+  </property>
+  <property>
+    <name>fs.azure.account.oauth2.client.id</name>
+    <value>[client-id]</value>
+  </property>
+  <property>
+    <name>fs.azure.account.oauth2.client.secret</name>
+    <value>[client-secret]</value>
+  </property>
+</configuration>
+```
+
+```shell md title="shell"
+java -jar utilities/target/utilities-0.1.0-SNAPSHOT-bundled.jar --datasetConfig my_config.yaml --hadoopConfig hadoop.xml
+```
+
+Running the above command will translate the table `people` in Iceberg or Hudi format to Delta Lake format. To validate
+the translation, you can list the directories in the table's data path. For this tutorial, `_delta_log` directory 
+should be present in the `people` table's data path at  `default/people/data/_delta_log`.
+
+### Step 3. Create a Shortcut in Fabric and Analyze
+This step creates a shortcut in Fabric to the Delta Lake table created in the previous step. The shortcut is a link to
+the table's data path in ADLS storage account.
+
+1. Navigate to your Fabric Lakehouse and click on the `Tables` and select `New Shortcut`.
+> ![Invoke new shortcut](/images/fabric/shortcut_1_1.png)
+
+2. In the `New Shortcut` dialog, select `Azure Delta Lake Storaget Gen2` as the `External Source`
+> ![Select external source](/images/fabric/shortcut_1_2.png)
+
+4. In the `New Shortcut` dialog, enter the `Connection settings` and authorize Fabric to access the storage account.
+> ![Enter connection settings](/images/fabric/shortcut_1_3.png)
+
+4. In the `New Shortcut` dialog, enter the `Shortcut settings` and click `Create`.
+> ![Enter shortcut settings](/images/fabric/shortcut_1_4.png)
+
+5. The shortcut is now created and the table is available for querying from Fabric.
+> ![Shortcut created](/images/fabric/shortcut_1_5.png)
+
+
+[^1]: Updated on 2023-11-16
```

Comment:
Can we remove the footnote? Just because it becomes tedious to track every change in the docs. In some pages I have done numerous changes whereas in one I haven't touched after the first push. Curious to know your thoughts.

Reply:
The purpose of the footnote is not to record style or minor content changes, but to indicate which version of the engine the tutorial was tested with. This is to inform the reader of possible changes in the future. For example, the steps with Fabric may be different in two months. 

If you have a better suggestion that is easier to manage or less confusing, please let me know.

Reply:
no, thats fair